specification/storage/resource-manager/Microsoft.Storage/2017-06-01/storage.json

```diff
@@ -1711,11 +1711,6 @@
           "type": "string",
           "x-ms-client-name": "ContentLanguage",
           "description": "The response header override for content language."
-        },
-        "rsct": {
```

Comment:
This is to just make sure that tool runs and outputs things in CI. I'll be reverting that commit as soon as review completes :)